core/remote/src/main/kotlin/com/trycatch/remote/di/NetworkModule.kt

```diff
@@ -34,20 +35,31 @@ import okhttp3.OkHttpClient
 import okhttp3.logging.HttpLoggingInterceptor
 import retrofit2.Retrofit
 import retrofit2.converter.kotlinx.serialization.asConverterFactory
+import javax.inject.Qualifier
 import javax.inject.Singleton
 
+
 @Module
 @InstallIn(SingletonComponent::class)
 internal object NetworkModule {
+    @Qualifier
+    @Retention(AnnotationRetention.BINARY)
+    annotation class CMC
+
+    @Qualifier
+    @Retention(AnnotationRetention.BINARY)
+    annotation class IPFS
```

Comment:
타겟 지정도 해두면 좋을 것 같아요~!

```suggestion
    @Qualifier
    @Retention(AnnotationRetention.BINARY)
    @Target(AnnotationTarget.FUNCTION, AnnotationTarget.VALUE_PARAMETER)
    annotation class CMC

    @Qualifier
    @Retention(AnnotationRetention.BINARY)
    @Target(AnnotationTarget.FUNCTION, AnnotationTarget.VALUE_PARAMETER)
    annotation class IPFS
```

Reply:
어노테이션의 네이밍도 살짝 의미가 불명확해 보이는 것 같기도해요..!

Reply:
말씀하신 대로 타겟을 명시해 두면 사용 범위가 더 명확해질 것 같아요. 그리고 어노테이션의 네이밍도 코드를 읽는 사람에게 의미를 더 잘 전달할 수 있도록 CMC 대신 CoinMarketCap으로, IPFS는 그대로 유지하거나 더 직관적인 이름으로 고민해 보겠습니다. 의견 반영하여 수정하겠습니다!